src/test/java/com/example/javaOjt/services/AuthorServiceTest.java

```diff
@@ -77,4 +80,60 @@ void getAuthorById_withBooks_false() {
     Mockito.verify(authorRepository, Mockito.never()).getAuthorWithBookDTOsById(targetId);
     Mockito.verify(authorRepository, Mockito.times(1)).findById(Mockito.any());
   }
+
+  @Test
+  void getAuthorsResponse_noParams() {
+    // Mocking the Authors response
+    List<Author> mockAuthors = List.of(new Author());
+    mockAuthors.get(0).setId(1);
+    mockAuthors.get(0).setName("Test Author1");
+    Mockito.when(authorRepository.getAuthorsList(AuthorSortBy.ID, null, null))
+        .thenReturn(mockAuthors);
+    // Call the service method
+    GetAuthorsResponse response = authorService.getAuthorsList(null, null, null);
+    // Assertions to verify the response
+    Assertions.assertNotNull(response);
+    Assertions.assertEquals(mockAuthors.size(), response.getAuthorsSummaryList().size());
+    Assertions.assertEquals(mockAuthors.getFirst().getId(),
+        response.getAuthorsSummaryList().getFirst().id());
+    Assertions.assertEquals(mockAuthors.getFirst().getName(),
+        response.getAuthorsSummaryList().getFirst().name());
+    Mockito.verify(authorRepository, Mockito.times(1)).getAuthorsList(AuthorSortBy.ID, null, null);
+  }
+
+  @Test
+  void getAuthorsResponse_ids() {
```

Comment:
修正に対して特に問題ないですが、テスト名だけ修正していただきたいです。

```suggestion
  void getAuthorsResponse_allParams() {
```